go/vt/mysqlctl/mysqld.go

```diff
@@ -688,7 +687,12 @@ func deleteTopDir(dir string) (removalErr error) {
 // executeMysqlCommands executes some SQL commands,
 // using the mysql command line tool.
 func (mysqld *Mysqld) executeMysqlCommands(connParams *sqldb.ConnParams, sql string) error {
-	return mysqld.executeMysqlScript(connParams, strings.NewReader(sql))
+	params, err := dbconfigs.WithCredentials(connParams)
```

Comment:
Please add a comment to this function as well then to better stress the difference between executeMysqlCommands and executeMysqlScript. Probably the comment on executeMysqlScript should be expanded as well to point out that if adding credentials is needed then executeMysqlCommands should be used, though then it becomes awkward due to difference in types of input arguments. :(